Added browserextension.dev to News

lindaicing opened the pull request:
Added browserextension.dev to News

Adjusted styling and spacing. Added spaces in CSS for formatting. 
Updated article heading sizes of articles:
<img width="1215" alt="Screen Shot 2020-07-24 at 8 24 35 PM" src="https://user-images.githubusercontent.com/38482675/88447748-b2438700-cdeb-11ea-8d5f-cba6785fc460.png">

News Section is still hidden, but when not hidden it looks like this:
<img width="1216" alt="Screen Shot 2020-07-24 at 8 22 57 PM" src="https://user-images.githubusercontent.com/38482675/88447726-77d9ea00-cdeb-11ea-98d2-0e5c9cbf645d.png">

CubeOfCheese commented on Website-and-Backend/src/main/resources/public/index.html:
```diff
@@ -114,9 +114,9 @@ <h3>About Ratings</h3>
-                    <p>News about Ethicli coming soon!</p>
-                    <p class="readmore"><a>Read More</a></p>
+                    <h3>BrowserExtension.dev</h3>
+                    <p>We've been featured by <a href="https://browserextension.dev" target="_blank">BrowserExtension.dev</a>, which is a website that discusses and explores new web extensions.</p>
```
You should link to our specific feature.

CubeOfCheese replied:
https://browserextension.dev/blog/ethicli/

CubeOfCheese replied:
Oh wait my bad nevermind

lindaicing commented:
> Maybe the other two should just be hidden. And then we could actually merge this and have it on our website. What do you think?
> Also, the menu color changing needs to be changed with this section being added back in

Ah shoot totally forgot about the menu, thank you!

lindaicing commented:
Ethicli in the News - Now shown
<img width="1303" alt="Screen Shot 2020-07-24 at 11 23 49 PM" src="https://user-images.githubusercontent.com/38482675/88450560-bc25b400-ce04-11ea-9050-5888dd92b616.png">

Featured Articles - Updated most recent article
<img width="1301" alt="Screen Shot 2020-07-24 at 11 24 12 PM" src="https://user-images.githubusercontent.com/38482675/88450569-c9db3980-ce04-11ea-841a-25f4af85e4fd.png">

Updated About Section to the phrase we came up with
<img width="476" alt="Screen Shot 2020-07-24 at 11 25 16 PM" src="https://user-images.githubusercontent.com/38482675/88450592-f000d980-ce04-11ea-996c-ec50c436eb2e.png">

Updated FAQ intro - Linked Intro Medium Article
<img width="884" alt="Screen Shot 2020-07-24 at 11 26 07 PM" src="https://user-images.githubusercontent.com/38482675/88450611-0e66d500-ce05-11ea-931b-f55dc124cfd4.png">
